jabgui/src/main/java/org/jabref/gui/maintable/MainTableTooltip.java

```diff
@@ -22,11 +25,49 @@ public class MainTableTooltip extends Tooltip {
     public MainTableTooltip(DialogService dialogService, GuiPreferences preferences, ThemeManager themeManager, TaskExecutor taskExecutor) {
         this.preferences = preferences;
         this.preview = new PreviewViewer(dialogService, preferences, themeManager, taskExecutor);
-        this.tooltipContent.getChildren().addAll(fieldValueLabel, preview);
+
+        fieldValueLabel.setWrapText(true);
+        fieldValueLabel.setMaxWidth(Double.MAX_VALUE);
+
+        tooltipContent.getChildren().addAll(fieldValueLabel, preview);
+        tooltipContent.setSpacing(5);
+
+        this.setMaxWidth(500);
+        this.setWrapText(true);
+
+        final double PREVIEW_WIDTH_PADDING = 16.0;
+        final double PREVIEW_HEIGHT_PADDING = 8.0;  // Padding to avoid bottom clipping of the preview
+        final double MIN_TOOLTIP_WIDTH = 200.0; // Minimum width of the tooltip to keep layout stable even with small content
```

Comment:
I don't understand. First line two spaces before `//` and second line one space. Be consistent. Either align both comments below each other or use one space only.